Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy

```diff
@@ -68,6 +68,41 @@ import groovy.json.JsonSlurper
  *  
  */
 
+class DeployableArtifact {
```

Comment:
Can we move the definition of the class to the bottom of the script and provide some inline documentation?

Reply:
Done